src/portal/src/app/project/repository/artifact/artifact-summary.component.html

```diff
@@ -15,19 +15,10 @@
     </div>
     <div class="title-block">
         <h2 class="custom-h2 center-align-items">
-            <img *ngIf="artifact?.type !== 'IMAGE';else elseBlock" class="artifact-icon" [title]="artifact.type"
-                 [src]="artifact.type | selectArtifactIcon" />
-            <ng-template #elseBlock>
-                <clr-tooltip>
-                    <div clrTooltipTrigger class="level-border">
-                        <img class="artifact-icon" [title]="artifact.type"
-                             [src]="artifact.type | selectArtifactIcon" />
-                    </div>
-                    <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
-                        <span>Docker and the Docker logo are trademarks or registered trademarks of Docker, Inc. in the United States and/or other countries.</span>
-                    </clr-tooltip-content>
-                </clr-tooltip>
-            </ng-template>
+            <div class="artifact-icon clr-display-inline-block">
+                <img *ngIf="artifact.icon && getIcon(artifact.icon)" class="artifact-icon" [title]="artifact.type"
```

Comment:
<img width="919" alt="Screen Shot 2020-08-11 at 3 08 59 PM" src="https://user-images.githubusercontent.com/8787455/89868377-47fd5580-dbe5-11ea-94e5-0b69998db35b.png">

Icon cannot display in your test env.